tslint-to-eslint-config.log

```diff
@@ -0,0 +1,31 @@
+6 ESLint rules behave differently from their TSLint counterparts:
```

Comment:
Do we need this checked in?

Reply:
Not necessary. I left it just in case we wanted to check it later